itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,267 @@ object CheckCommand "printer_health" {
 	vars.printer_health_hostname = "$check_address$"
 	vars.printer_health_mode = "supplies-status"
 }
+
+template CheckCommand "generic-thola-check-command" {
```

Comment:
Do you import this template more than once? If no, why isn’t it inlined?

Reply:
We wanted to import it also for the thola-server command but accidentally imported the check command template. So then it is used twice.